src/BTCPayServer.Lightning.Common/PayResponse.cs

```diff
@@ -5,6 +5,7 @@ namespace BTCPayServer.Lightning
     public enum PayResult
     {
         Ok,
+        Unknown,
```

Comment:
Shouldn't it be `Pending`?

Reply:
I had that at first, but see the end of the Pay methods: we need to return something there as well and it didn't seem right to have that state as Pending, because it'd be Unknown. 

It's an edge case that'll most likely not occur (keysend + timeout) and I didn't want to introduce two new states, so Unknown seemed like a good compromise.